manageiq-operator/pkg/helpers/miq-components/cr.go

```diff
@@ -298,26 +354,38 @@ func ManageCR(cr *miqv1alpha1.ManageIQ, c *client.Client) (*miqv1alpha1.ManageIQ
 		cr.Spec.HttpdAuthenticationType = httpdAuthenticationType(cr)
 		cr.Spec.HttpdImageNamespace = httpdImageNamespace(cr)
 		cr.Spec.HttpdImageTag = httpdImageTag(cr)
+
+		privileged, err := PrivilegedHttpd(cr.Spec.HttpdAuthenticationType)
```

Comment:
Should this be in the httpdImage method?

Reply:
meh, if we do, then httpdImage would need to return string, error and check for that here, so it won't be the one liners like all others here.

Reply:
Why would `httpImage` need to return error?

Reply:
httpdImage would need to call PriviledgedHttpd to get that privileged value for its logic and that method could return an error.

Reply:
Wouldn't we ignore the error in the `httpdImage`function just like we're ignoring it here?

Reply:
I think the "validation" that happens in `PrivilegedHttpd` which could return an error should ideally be part of the CRD Schema (although that doesn't appear to be supported) or our custom validation logic.